src/genie/libs/parser/iosxr/tests/ShowImDampeningIntf/cli/equal/golden_output2_expected.py

```diff
@@ -0,0 +1,31 @@
+expected_output = {
+                    'interface': {
+                        'TenGigE 0/1/0/0': {
+                            'currently_suppressed': 'yes',
+                            'dampening_status': 'enabled',
+                            'half_life': 1,
+                            'max_supress_time': 4,
+                            'penalty': 1625,
+                            'protocol': {
+                                'ipv4': {
+                                    'protocol_capsulation': 'ipv4',
```

Comment:
these ones has `protocol` level at line 9. so I believe we don't need prefix `procotol_`.
```
'protocol': {
    'ipv4': {
        'capsulation': 'ipv4',
        'penalty': 1615,
        'suppression': 'NO',
        'suppression_remaining_secs': 22,
        'underlying_state': 'Down',

Reply:
Sure, I'll do that.. :)

Reply:
```
                            'index': {
                                1: {
                                    'capsulation': 'ipv6',
                                    'penalty': 1625,
                                    'protocol': 'ipv6',
                                    'suppression': 'YES',
                                    'suppression_timer': 42,
                                    'underlying_state': 'Down'
                                },
```
can I ask you change from `suppression_timer` to `suppression_remaining_sec`? better to have keyword `remaining` and also unit `secs`.

Reply:
Sorry I missed that from the first round, I'll make that change.

Reply:
I've made that change and uploaded an amended version for that. Thanks.

Reply:
confirmed the change